salt_stack/salt/top.sls

```diff
@@ -7,8 +7,6 @@ base:
     'vault*':
         - vault.server
         - boss-tools.bossutils
-        - scalyr
```

Comment:
Removing the scalyr formulas will turn off logging completely for **all** stacks including production.

Reply:
Good point, I'll add this back and re-test

Reply:
👍

Reply:
Why remove scalyr in general?